sympy/matrices/tests/test_matrices.py

```diff
@@ -2981,6 +2981,7 @@ def test_issue_8207():
     assert d == b[0, 0]
     assert e == 0
 
+
```

Comment:
```suggestion
```
IMHO, this change relevant for this PR as `matrices` isn't in the scope of the issue.